src/main/java/com/cb/controllers/LoginController.java

```diff
@@ -34,31 +33,28 @@
 
     @RequestMapping(value = "/loginuser", method = RequestMethod.POST)
     public String loginUser(HttpServletRequest req, Map<String, String> model, @ModelAttribute("userDAL") UserBL userBL, Model p, Model c) {
-        List<UserDAL> allEmails = userService.getUserByEmail(userBL.getEmail());
-        List<UserDAL> allPasswords = userService.getUserByPassword(userBL.getPassword());
-        String userName = "";
-        for (UserDAL items: allEmails
-        ) {
-            userName = items.getUserName();
-        }
+        int userExist = userService.getUserByEmailAndPassword(userBL);
 
-        if (allEmails.size() > 0 && allPasswords.size() > 0) {
+        if (userExist == 1) {
+            String userName = userService.getUserNameByEmail(userBL);
```

Comment:
We need to implement situation when user already created a character. So if party and member exist, then we need to move to another (main) page.